src/OrchardCore.Modules/OrchardCore.Taxonomies/Views/LocalizationPart.Edit.cshtml

```diff
@@ -0,0 +1,57 @@
+
+<input asp-for="LocalizationSet" type="hidden" />
+<input asp-for="Culture" type="hidden" />
+<div class="form-group content-localization" id="localizationPartEdit">
+    <div class="card">
+        <div class="card-header" id="contentLocalizationToggle">
+            <button class="d-flex align-items-center justify-content-between btn btn-link collapsed" type="button" data-toggle="collapse" data-target="#contentLocalizationCard" aria-expanded="true" aria-controls="collapseOne">
+                @T["Current Culture: {1} | {0}", Model.CultureInfo.Name, Model.CultureInfo.DisplayName]
+                <i class="fa fa-angle-down" aria-hidden="true"></i>
+            </button>
+        </div>
+
+        <div id="contentLocalizationCard" class="collapse" aria-labelledby="contentLocalizationToggle" data-parent="#localizationPartEdit">
+            <div class="card-body">
+                @if (!Model.ContentItemCultures.Any())
+                {
+                    <p>@T["Your site does not have any cultures. Current System culture is {1} | {0}", CultureInfo.InstalledUICulture.Name, CultureInfo.InstalledUICulture.DisplayName]</p>
+                    <p><a asp-route-area="OrchardCore.Settings" asp-controller="Admin" asp-action="Index" asp-route-groupId = "localization">@T["Add or remove supported cultures for the site"]</a></p>
+                }
+                else
+                {
+                    @if (!string.IsNullOrEmpty(Model.LocalizationSet))
+                    {
+                        <div class="list-group">
+                            @foreach (var culture in Model.ContentItemCultures)
+                            {
+                                if (!string.IsNullOrEmpty(culture.ContentItemId))
+                                {
+                                    <a class="list-group-item list-group-item-action" asp-action="Edit" asp-controller="Admin" asp-route-area="OrchardCore.Contents" asp-route-contentItemId="@culture.ContentItemId" title=@T["Edit {0}", culture.Culture.DisplayName]><i class="fa fa-edit" aria-hidden="true"></i> @culture.Culture.DisplayName | @culture.Culture.Name </a>
+                                }
+                                else
+                                {
+                                    @if (@Context.Request.RouteValues["containerContentItemId"] != null)
+                                    {
+                                        <a class="list-group-item list-group-item-action" data-url-af="UnsafeUrl" asp-action="Localize" asp-controller="Admin" asp-route-area="OrchardCore.Taxonomies" asp-route-containerContentItemId="@Context.Request.RouteValues["containerContentItemId"]" asp-route-contentItemId="@Model.LocalizationPart.ContentItem.ContentItemId" asp-route-targetCulture="@culture.Culture.Name" title=@T["Create {0}", culture.Culture.DisplayName]><i class="fa fa-plus" aria-hidden="true"></i> @culture.Culture.DisplayName | @culture.Culture.Name</a>
```

Comment:
Here I'm changing the AreaName so that it triggers the proper controller action. I tried adding a dynamic AreaName based on the current Request previously and it was not resolving the proper Route/Action. But when I'm overriding this shape view inside the Taxonomy module it returns the proper Route/Action.